cmd/release/cmd/tag.go

```diff
@@ -62,6 +128,13 @@ func init() {
 	tagCmd.AddCommand(k3sTagSubCmd)
 	tagCmd.AddCommand(rke2TagSubCmd)
 
+	dryRun = tagCmd.PersistentFlags().BoolP("dry-run", "d", false, "Dry run")
```

Comment:
Maybe move this persistent flag to root.go

Reply:
I had given that some thought but not all commands will have/need dry-run.